arclight-neoforge/src/main/java/io/izzel/arclight/neoforge/mixin/core/world/item/ItemStackMixin_NeoForge.java

```diff
@@ -38,8 +38,13 @@ public abstract class ItemStackMixin_NeoForge implements ItemStackBridge, IItemS
     // @formatter:on
 
     @Decorate(method = "hurtAndBreak(ILnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/entity/LivingEntity;Ljava/util/function/Consumer;)V",
-        at = @At(value = "INVOKE", target = "Lnet/minecraft/world/item/enchantment/EnchantmentHelper;processDurabilityChange(Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/item/ItemStack;I)I"))
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/world/item/enchantment/EnchantmentHelper;processDurabilityChange(Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/item/ItemStack;I)I"))
     private int arclight$itemDamage(ServerLevel serverLevel, ItemStack itemStack, int i, @Local(ordinal = 0) LivingEntity damager) throws Throwable {
+        // Added null check for itemStack
```

Comment:
Would you mind figuring out which method passed a null as `ItemStack` into this method?  
In Minecraft modding, these should all be replaced by `ItemStack,EMPTY`. If it happened in NeoForge, perhaps it was their bug.

Reply:
Sure i will look into it asap